### ISSUE TITLE
Feature request: querying device model name

### ISSUE THREAD

**Issue by dsvdmeer** [star-micronics/StarXpand-SDK-iOS#28]
## Is your feature request related to a problem? Please describe.
For Bluetooth printers there currently seems to be no way to get a printer's model name. We use this information in our UI to show discovered printers. In the StarPRNT SDK this is possible using `getFirmwareInfo()`, but there is no equivalent in the StarXpand SDK (or I can't find it?). The `information?.modal` property always returns `unknown`.

## Describe the solution you'd like
A function similar to the `getFirmwareInfo()` function in the StarPRNT SDK. We actually only use the "ModelName" field from that and ignore that firmware version, so a `getModelName()` function would also suffice.

## Describe alternatives you've considered
We're currently using the StarPRNT SDK and it works, but we'd like to update to this new SDK. At the moment the lack of this feature stops us from doing so.

## Additional context
I believe that such a feature would also be beneficial for non-bluetooth printers because it returns a human-readable model name. The `StarPrinterModel` enum does not provide these human-readable names.

**Comment by Tatsuki-Yamamoto2731** [comment]
@dsvdmeer 
Do you use the following models?
- TSP650II
- TSP700II
- TSP800II

For Bluetooth, these models cannot be obtained model names when discovered.
But once open() is executed, you can obtain the printer model name from StarPrinterInformation.model.

I have confirmed that TSP650II is retrieved as "model: unknown" during discovery and can be retrieved as "model: tsp650II" after open() run.
Could you please try it?

**Comment by dsvdmeer** [comment]
Yes we use the TSP650II. I had not tried to first call `open()`, thanks for that suggestion!

**Comment by Tatsuki-Yamamoto2731** [comment]
@dsvdmeer Thank you for your comment.
Unfortunately, we don't have such a function now, but we understand your needs, so we will consider based on your feedback.
